topics/materials-science/metadata.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: materials-science
+type: use
+title: Materials Science
+summary: Modelling, simulation and analysis for the material science domain.
+docker_image: ''
```

Comment:
Should the docker image be referenced here?

Reply:
I'll add a refernce to https://quay.io/organization/galaxy in the same format as the other topics.

I had assumed that the image will be pushed automatically as part of the CI, but it appears the `docker-build.yaml` workflow hasn't run since March 2021 so maybe not. Either way, possibly something to raise when making a PR against the upstream repo.